api/groups/addressGroup.go

```diff
@@ -20,6 +20,7 @@ const (
 	getAccountPath            = "/:address"
 	getBalancePath            = "/:address/balance"
 	getUsernamePath           = "/:address/username"
+	getCodeHashPath						= "/:address/code-hash"
```

Comment:
wrong alignment

Reply:
You are right. I indented with tabs instead of spaces and didn't remark it in my editor